include/mgp.hpp

```diff
@@ -2486,6 +2532,24 @@ inline bool Map::operator==(const Map &other) const { return util::MapsEqual(ptr
 
 inline bool Map::operator!=(const Map &other) const { return !(*this == other); }
 
+inline const std::string Map::ToString() const {
+  std::string return_string{"{"};
+  const size_t map_size = Size();
+  if (map_size == 0) {
+    return "{}";
+  }
+  size_t i = 0;
+  for (const auto &[key, value] : *this) {
+    if (i == map_size - 1) {
+      return_string.append(std::string(key) + ": " + value.ToString() + "}");
+      break;
+    }
+    return_string.append(std::string(key) + ": " + value.ToString() + ", ");
+    ++i;
+  }
+  return return_string;
```

Comment:
is it possible to use `fmt` library here?

Reply:
I wouldn't introduce fmt library here if not necessary